services/src/contexts/mod.rs

```diff
@@ -56,9 +63,30 @@ pub trait Context: 'static + Send + Sync + Clone {
 
     fn execution_context(&self, session: Self::Session) -> Result<Self::ExecutionContext>;
 
+    fn task_manager(&self) -> &TaskManager;
+
     async fn session_by_id(&self, session_id: SessionId) -> Result<Self::Session>;
 }
 
+#[derive(Clone)]
+pub struct TaskManager {
+    plot_executor: Arc<Executor<WorkflowId, Result<Value>>>,
+}
+
+impl TaskManager {
+    pub fn plot_executor(&self) -> &Executor<WorkflowId, Result<Value>> {
+        self.plot_executor.as_ref()
+    }
+}
+
+impl Default for TaskManager {
+    fn default() -> Self {
+        TaskManager {
+            plot_executor: Arc::new(Default::default()),
+        }
+    }
+}
+
```

Comment:
Can't we move this to the pro-part and into the pro context?

Reply:
30a6c58dfaebf7bfc8c5bb4501ed48684c9a64a8
7cd7fd0d2733776901d4ee8aebdcf8191cbc5fe1